cloudbuild.yaml

```diff
@@ -32,7 +32,18 @@ steps:
   args:
   - -c
   - mvn --no-transfer-progress -e install -Dlicense.skip=true -Dspotless.apply.skip=true
-  waitFor: ['-'] 
+  waitFor: ['-']
+
+- name: 'gcr.io/cloud-builders/curl'
+  id: 'Upload coverage reports to codecov.io'
+  entrypoint: bash
+  args: ['-c', 'bash <(curl -s https://codecov.io/bash)']
```

Comment:
Do we also see the line-by-line coverage in the review tool of GitHub? I mean while reviewing the changes, see whether the new/modified lines are covered or not.

Reply:
yes, the link provided by codecov-commenter should have that.